src/libs/ValidationUtils.js

```diff
@@ -357,46 +357,25 @@ function isValidRoutingNumber(number) {
 }
 
 /**
- * Checks if each string in array is of valid length and then returns true
- * for each string which exceeds the limit.
+ * Checks if each provided string includes any commas or semicolons
  *
- * @param {Number} maxLength
  * @param {String[]} valuesToBeValidated
  * @returns {Boolean[]}
  */
-function doesFailCharacterLimit(maxLength, valuesToBeValidated) {
-    return _.map(valuesToBeValidated, value => value && value.length > maxLength);
+function doesContainCommaOrSemicolon(valuesToBeValidated) {
```

Comment:
NAB: The name makes me wonder what the subject is. What contains a comma or semicolon? I also think mapping through the values feels a bit overkill.

How about `isValidDisplayName` and we check if the value contains comma or semi colon, and we call it once for each name?

Reply:
Yeah, that sounds good actually. That feels more straightforward. I'll update.

Reply:
Updated.